source/common/quic/envoy_quic_client_session.cc

```diff
@@ -1,5 +1,6 @@
 #include "source/common/quic/envoy_quic_client_session.h"
 
+#include "source/common/common/empty_string.h"
```

Comment:
revert

Reply:
Reverted in commit 34ba6e20b006a8f73181b52d8505965d0327fc31.